tests/unit/tls_test.cc

```diff
@@ -65,8 +65,8 @@ static std::string certfile(const std::string& file) {
 
 using namespace seastar;
 
-template <typename Func>
-auto with_https_server(Func&& func) {
+template <typename T, typename Func>
+auto with_https_server_and(T&& v, Func&& func) {
```

Comment:
Why not rebase this change into the first one?

Reply:
i wanted to reduce the pain of reviewing because of reindent only changes. will fold this change into the "[tls_test: use dedicated https server for testing](https://github.com/scylladb/seastar/pull/1348/commits/9b339d9585abdf23c5505b45100af9bf03c4caa7)" commit.

Reply:
fair enough.